src/sink.h

```diff
@@ -175,6 +175,11 @@ class Sink : public cyclus::Facility  {
                              "accept at each time step"}
   double capacity;
 
+  #pragma cyclus var {"default": "", "tooltip": "requested composition", \
+                      "doc": "name of recipe to use for material " \
+                             "requests"}
```

Comment:
Updated!


> On Feb 11, 2015, at 10:40 AM, Anthony Scopatz <notifications@github.com> wrote:
> 
> In src/sink.h <https://github.com/cyclus/cycamore/pull/282#discussion_r24509727>:
> 
> > @@ -175,6 +175,11 @@ class Sink : public cyclus::Facility  {
> >                               "accept at each time step"}
> >    double capacity;
> >  
> > +  #pragma cyclus var {"default": "", "tooltip": "requested composition", \
> > +                      "doc": "name of recipe to use for material " \
> > +                             "requests"}
> Also some documentation that "an empty string (default) means to accept everything." That isn't intuitive from the interface.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/cyclus/cycamore/pull/282/files#r24509727>.
> 

***************************************************************
Meghan McGarry, PhD ~ http://mbmcgarry.github.io/ <http://mbmcgarry.github.io/> 
Postdoctoral Fellow, Consortium for Verification Technology
Computational Nuclear Engineering ~ http://cnerg.engr.wisc.edu <http://cnerg.engr.wisc.edu/>
Engineering Physics ~ UW-Madison ~ 608-263-6974
***************************************************************